experimental/framework/data-objects/src/signaler/signaler.ts

```diff
@@ -67,6 +68,8 @@ export interface ISignaler extends IEventProvider<IErrorEvent> {
  * Duck type of something that provides the expected signalling functionality:
  * A way to verify we can signal, a way to send a signal, and a way to listen for incoming signals
  * @internal
+ * @privateRemarks
+ * There is no use external to package and export can be removed once breaking changes are permitted.
```

Comment:
As an experimental package, breaking changes are allowed at any time.

Reply:
Yes - be we are doing absolutely zero breaking changes until at least after 2.2. While this package is experimental, customers use it in production.